add and update files per PRs 5023, 5004, 5000, 4995, 4988

peiying2 opened the pull request:
- 5018: BATM for de only
- 5004: new plugin check
- 5000: technology
- 4995: footer
- 4988: send tabs and fx index.

peiying2 commented:
NOTE: 
- 5018: I couldn't extract the file though Craig had no issue, but luckily there are 3 strings.  I don't know what the proper file name should be.  He suggests to use new/batm/machine.  The demo URL/Link needs to up update, and potentially file moves around. There is a bit of urgency 
** PRs below contain lot of strings leverageable from other files. Questions: when we group "shared" strings, does it need to be under the same subpages. Under Firefox or IH or mozorg.  Or does it matter?  For now, I leave them where they are, and removed some that are obvious. 
- 5004: plugin can leverage from the old plugin and some shared strings.
- 5000: technology - this contains so many shared strings from the past. I left them as they are.
- 4995: footer some shared strings from from Fx shared files.
- 4988: has shared strings from the main.lang, and some within the Fx subpages but  spread out in several pages.
-

flodolo commented on en-US/main.lang:
```diff
@@ -670,20 +670,49 @@ Mozilla Firefox Web Browser
+Beta, Nightly, Developer Edition
+
+
+# String is obsolete. Do NOT remove.
```
Not true, it's used a few lines below (line 42 in the PR)

flodolo commented on en-US/mozorg/plugincheck-update.lang:
```diff
@@ -0,0 +1,64 @@
+
+
+# HTML page decription
+# NPAPI stands for Netscape Plugin Application Programming Interface. It is an API that allows plugins (more specifically, browser extensions) to be developed for web browsers.
```
Plugins and browser extensions are not the same thing. I would drop the second part of the doc

flodolo commented on en-US/mozorg/plugincheck-update.lang:
```diff
@@ -0,0 +1,64 @@
+
+
+;Hungry for more?
+Hungry for more?
```
We'll need to import the newsletter strings from firefox/shared.lang

flodolo commented:
One more
https://github.com/mozilla/bedrock/pull/5023

peiying2 commented on en-US/firefox/new/batm.lang:
```diff
@@ -0,0 +1,15 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/new/batm/
```
The link is still craig's personal test server. I am fine if we remove it from this PR. Oliver is on vacation, i can wait until craig's work is merged to work on this one.

peiying2 commented:
- Removed # 5018 (batm.lang) from the PR since Oliver is on vacation until next Monday. Hopefully all is clearer by Friday.
- Moved sendto_tab file to the proper home.
- added PR 5023

flodolo commented on en-US/main.lang:
```diff
@@ -670,20 +670,48 @@ Mozilla Firefox Web Browser
 Download Firefox Web Browser
 
 
+# String is obsolete. Do NOT remove.
```
They're not obsolete yet, because they're displayed on the page. They might become later on, but it's too early too add such comment.

peiying2 commented:
@flodolo on the sendto-tabs.lang, I am so lost at where the file should be.  I will fix this one before i muck up the langchecker file again. Here are my clues. I need to get this right as we have more files like this here or elsewhere.  
- .../firefox/features/send-tabs.html - template
- add_lang_files "firefox/shared.lang - string extraction ends up in this file.  
- extends "firefox/features/base.html (do i need to read this line?) - I assume the file name is /firefox/features/send-tabs.lang, under the fx/feature/... structure.

flodolo commented:
Have you checked the docs we have? Or, have you checked similar templates in /features and see how they are mapped to a lang file?

Last question: if the template is called firefox/features/send-tabs.html, can you describe the reasoning that gets you to create a lang file called firefox/features/sendto-tabs.lang?

peiying2 commented:
Doh.... sendto-tabs vs send-tabs
